CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - [core] added support for several international keyboard layouts
 - [core] implemented auto-detection of keyboard layout based on pressed keys
 - [core] added command to manually choose a keyboard layout
+- [plugin] 'Hosted mode' extracted in `plugin-dev` extension
```

Comment:
please move it to 0.8.0 as a breaking change

to my knowledge plugin system already used in several products running in production with customizations, should not be a surprise that we move some APIs

Reply:
Done